src/components/DiffView/index.tsx

```diff
@@ -201,7 +201,11 @@ export class DiffViewBase extends React.Component<Props> {
 
     const allWidgets: WidgetMap = {};
 
-    for (const change of getAllHunkChanges(hunks)) {
+    // We only want widgets for delete, insert and normal changes, not eofnl changes.
+    const changesAllowingWidgets = getAllHunkChanges(hunks).filter(
+      (change) => change.isDelete || change.isInsert || change.isNormal,
```

Comment:
Why deletes? It would be nice if you could reuse `changeCanBeCommentedUpon()` here.

Reply:
Ah, nevermind, it's because of linter messages.